pkg/rhttp/datatx/utils/download/download.go

```diff
@@ -104,6 +104,9 @@ func GetOrHeadFile(w http.ResponseWriter, r *http.Request, fs storage.FS, spaceI
 			return
 		}
 	}
+	if content == nil {
```

Comment:
why this? we have already checked that `err` is not `nil` and we expect that the content is not nil. If the callee sets a content to `nil` this should be resolved in the driver.